README.rst

```diff
@@ -555,7 +556,7 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **simplified_null_return**
-   | A return statement wishing to return nothing should be simply "return".
+   | A return statement wishing to return "void" should not return "null".
```

Comment:
and the new one:
`A return statement wishing to return "void" should not return "null".`

is suggesting that we do check the willingness